pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/metrics/ApexMetrics.java

```diff
@@ -37,8 +37,11 @@ public static ApexMetricsFacade getFacade() {
     }
 
 
-    /** Resets the entire data structure. Used for tests. */
-    static void reset() {
+    /**
+     * Resets the entire data structure.
+     * This needs to be done in case PMD is executed multiple times within one JVM run.
+     */
+    public static void reset() {
```

Comment:
Maybe, a better solution wrt to visibility would be to move the LanguageMetricsProvider out of ApexHandler, and into this package. ApexMetrics will probably not change for 7.0 (apart from this method), while LanguageMetricsProvider is still experimental and we can remove the new API freely in 7.0